### DISCUSSION TITLE
Recent Imitation Learning algorithms such as ACT and Diffusion Policies

### DISCUSSION THREAD

**Discussion by SakethTecholution** [isaac-sim/IsaacLab#1356] [Q&A] [answered]
Hello IsaacLab Team,
I have been trying to use the default imitation learnging algorithms but it doesnt work that good. So is there any chance that recent developements and State of Art imitation learnings algorithms like ACT or Diffusiion can be integrated to Isaac Lab. If yes can you provide some ways to work on it

**Comment by SakethTecholution** [comment]
cc
@Mayankm96 @Dhoeller19 @hhansen-bdai @masoudmoghani

**Comment by StrainFlow** [comment, marked as answer]
Here are the libraries we support: https://isaac-sim.github.io/IsaacLab/main/source/overview/reinforcement-learning/rl_frameworks.html

Some of these four supported frameworks support multiple algorithms, and so one of them might do what you need. 

If none of these work for you, we welcome contributions and would love to have users like yourself add support for additional RL frameworks. Here are the contribution guidelines: https://isaac-sim.github.io/IsaacLab/main/source/refs/contributing.html
